core/vmpu/src/mpu_kinetis/vmpu_kinetis_mpu.c

```diff
@@ -28,7 +28,7 @@
 #include "configurations.h"
 
 /* The K64F has 12 MPU regions, however, we use region 0 as the background
- * region with `UVISOR_TACL_BACKGROUND` as permissions.
+ * region with `UVISOR_TACL_DEBUGGER_BACKGROUND` as permissions.
  * Region 1 and 2 are used to unlock Application SRAM and Flash.
```

Comment:
This is not true anymore for regions 1 and 2, right?

Reply:
fixed docs